OneCommunityGlobal/HighestGoodNetworkApp
# Zijie lint fix reducer phase 1

# Description
This is lint fix for the actionltemsReducer.js, allProjectsReducer.js, allTasksReducer.js, allTeamsReducer.js, allTimeEntriesReducer.js from src/reducers file

![CleanShot 2023-10-18 at 21 41 06@2x](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/46791134/142aba11-cf94-410b-aad0-3b203344f1fa)




## How to test:
1. check into current branch
2. do `npm install` and `npm run start:local` to run this PR locally
3. log as admin user

(Please be sure to record a video)
5. **Verify that the project/task/teams/time entries reducer is working normally.**

~~6. Delete .eslintignore file in the root: `rm .eslintignore`
7. `npx eslint src/reducers/actionItemsReducer.js`
8. `npx eslint src/reducers/allProjectsReducer.js`
9. `npx eslint src/reducers/allTasksReducer.js`
10. `npx eslint src/reducers/allTeamsReducer.js`
11. `npx eslint src/reducers/allTimeEntriesReducer.js`
12. `git restore .`~~
